VERSION

```diff
@@ -1 +1 @@
-0.1.1
\ No newline at end of file
+0.1.1-mearns

```

Comment:
Yeh, I didn't want have my fork version out there with the same version number as yours.